pkg/outputs/nats_outputs/jetstream/jetstream_output.go

```diff
@@ -527,6 +529,17 @@ func (n *jetstreamOutput) subjectName(m proto.Message, meta outputs.Meta) (strin
 	switch n.Cfg.SubjectFormat {
 	case subjectFormat_Static:
 		sb.WriteString(n.Cfg.Subject)
+	case subjectFormat_SubjectTargetSub:
+		sb.WriteString(n.Cfg.Subject)
+		sb.WriteString(".")
+		err := n.targetTpl.Execute(sb, meta)
+		if err != nil {
+			return "", err
+		}
+		sb.WriteString(".")
+		if sub, ok := meta["subscription-name"]; ok {
+			sb.WriteString(sub)
+		}
```

Comment:
This assumes that a `subject` is always present.
Can we change this format to `target.subscription` and include `subject` only if `!= ""` ?
I can update the other formats to follow the same principle.

Reply:
That's a good idea, I'll update it.